packages/js/manifests/wrap/src/formats/wrap.info/index.ts

```diff
@@ -6,26 +6,37 @@
  */
 
 import {
-  WrapManifest as WrapManifest0_1_0
+  WrapManifest as WrapManifest_0_1_0,
+  Abi as WrapAbi_0_1_0_0_0_1
 } from "./0.1.0";
 
 export {
-  WrapManifest0_1_0,
+  WrapManifest_0_1_0,
+  WrapAbi_0_1_0_0_0_1
 };
 
 export enum WrapManifestVersions {
   "0.1.0" = "0.1.0",
-}
+};
+
+export enum WrapAbiVersions {
+  "0.1.0" = "0.0.1",
```

Comment:
What's the reason for making the value different? Would it make sense to -for now- keep consistency between these?

Reply:
I don't know it was like this in your original commit

Reply:
okkk my bad then :P
```suggestion
  "0.1.0" = "0.1.0",
```

Reply:
I'll update it then.